doc/source/user_guide/mapdl.rst

```diff
@@ -618,8 +619,8 @@ using the :attr:`Mapdl.chain_commands <ansys.mapdl.core.Mapdl.chain_commands>` a
 
 The execution time using this approach is generally 4 to 10 times faster than running
 each command individually. You can then view the final response of
-the chained commands with the :attr:`Mapdl.last_response
-<ansys.mapdl.core.Mapdl.last_response>` attribute.
+the chained commands with the
+:attr:`Mapdl.last_response <ansys.mapdl.core.Mapdl.last_response>` attribute.
```

Comment:
I will be nice to leave some of these "broken" between to lines, so we can make sure the regex used works. ;)

Reply:
I will revert one of them.